src/ert/gui/tools/manage_cases/case_init_configuration.py

```diff
@@ -232,7 +233,7 @@ def addShowCaseInfo(self):
 
         layout.addLayout(row1)
 
-        self._case_info_area = QTextEdit()
+        self._case_info_area = QTextEdit(objectName="html_text")
```

Comment:
Why did you set this name specifically?
Is this being used somewhere i can't see, or was intended to be used?

Reply:
It is used in the new test to select this specific `QTextEdit` widget. 

```python
html_edit = tool.findChild(QTextEdit, name="html_text")
```

The name can be removed but if in the future somebody will add another anther QTextEdit the test will fail.

Reply:
No, keep that as is. I must have overlooked the test when searching. :+1: